checksec

```diff
@@ -764,6 +763,18 @@ kernelcheck() {
     echo_message "\033[31mNone\033[m\n" "None," " randomize_va_space='none'" '"randomize_va_space":"none",'
   fi
 
+  echo_message "  NX protection:                          " "" "" ""
+  if (command_exists dmesg) && (root_privs); then
```

Comment:
Looking closer at this I think [nxcheck](https://github.com/slimm609/checksec.sh/blob/348fbc9601fbf06499f3b7de560d1095310062e7/checksec#L691) function already does the job in easier way. Perhaps it's better to call `nxcheck` again and remove `nx_protection` code completely.

Reply:
The `nxcheck` checks if the cpu supports NX.
This checks if the kernel has NX check active.
Is this always equivalent?

Reply:
It should be equivalent unless someone passed `noexec=off` in `cmdline`. If `checksec --file` treats NX cpu support as NX available then why `checksec --kernel` should show something different?